lib/chewy/type/import/journal_builder.rb

```diff
@@ -31,7 +31,7 @@ def entries(action, objects)
             index_name: @type.index.derivable_name,
             type_name: @type.type_name,
             action: action,
-            references: identify(objects).map(&:to_json).map(&Base64.method(:encode64)),
+            references: identify(objects).map { |item| Base64.encode64(::Elasticsearch::API.serializer.dump(item)) },
```

Comment:
Didn't get how this one is related

Reply:
That's the serialization optimization mentioned above. This uses the serializer Elasticsearch decided to use, instead of the slow default one.